### DISCUSSION TITLE
Ошибка при открытии файла в Onlyoffice

### DISCUSSION THREAD

**Discussion by cartlife** [totumonline/totum-ru-issues-and-discussions#593] [ОШБК!] [answered]
Подскажите, пожалуйста, что может быть. При открытии на чтение или редактирование файла появляется ошибка "you are trying to perform an action you do not have rights for."
![2024-09-01](https://github.com/user-attachments/assets/bc12162e-95c1-46e8-8a6d-304ba98140a6)
Устанавливал на сервер по инструкции (уже три раза пробовал), токен перепроверял много раз, пробовал с ip и без него в таблице Настройки. По адресу приветственная страница открывается.
Гуглил ошибку, но ничего внятного не нашел.

**Comment by ttm-alexeypolunin** [comment]
OF у себя что-то обновил и нам нужно по новой разбираться, что они там сделали и почему не поддерживают обратную совместимость.

Исправлять мы будем это в V5 и когда по срокам не знаю.

**Reply by ttm-alexeypolunin** [reply]
Вообще думаю раз такое дело, мы ща его сразу только при наличии платной лицензии сделаем доступным.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Исследование вопроса показало, что ошибка в правах базы данных. Инструкция, которую они предоставляют для debian-подобный систем перестала быть валидной из-за поставки с новыми версиями отличающейся версии postgresql, поэтому grant привелегий пользователю onlyoffice больше не работает с их скриптом. Стал обязательным ALTER DATABASE OWNER.

Мы откорректировали инструкцию у нас на сайте и проверили, что установка работает (на U24.04).

Но наши планы убирать интеграцию под пайволл остались неизменными.
